### ISSUE TITLE
Service interface to service health

### ISSUE THREAD

**Issue by yblatti** [easySDI/easySDI#488]
**Original author :** Larsen de Martino Jorgen

A service interface to service health would be much appreciated - 

To be used to show service health on other platforms than JOOMLA.

 - Not sure if it already done -

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

Hello Jorgen,

Yes, we have some "piece of code" to do this, but it's relatively simple. We created a Joomla! plugin that request the MONITOR API (java servlet) to display the current status of a job (statuts available, not available, etc.) 

See here : http://forge.easysdi.org/projects/monitor/wiki/Frontend_plugin

It's done within Joomla! but could be done outside Joomla! as well if the API is well and enough documented -> we did the documentation for the first version before your contribution (see here : http://forge.easysdi.org/projects/monitor/wiki) so documentation should be updated with new features you added.

A "service health" view with historic data (what we have in report tab in the backend) could be done and proposed via the API by extended it. In that way, any third application could directly request this API to get this health view.

_By the way, you asked what is "public/private" statuts for a job in another issue -> we introduced this status for each job to distinguish jobs that can be requested by anyone without authentication (via the API) and otehrs jobs that can only be requested with the admin account (via the API). The idea was that we would like to monitor services but not show results publicly. Hope it's clear..._
